File: HELLO_ANDROID01/app/src/main/java/kr/co/aiai/app/MainActivity02.java
package kr.co.aiai.app;

import android.os.Bundle;
import android.util.Log;
import android.view.View;
import android.widget.Button;
import android.widget.TextView;

import androidx.appcompat.app.AppCompatActivity;

public class MainActivity02 extends AppCompatActivity {
    TextView tv = null;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main02);
        Log.d("aiai","안녕");
        Button btn = findViewById(R.id.btn);
        tv=findViewById(R.id.tv);
        btn.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                myclick();
            }
        });
    }
    void myclick(){
        Log.d("tookeon","myclcik");
        int cnt=Integer.parseInt((String)tv.getText());
        cnt--;
        tv.setText(String.valueOf(cnt));
    }
}

File: HELLO_AI4/ANDROID/app/src/main/java/kr/co/aiai/myapp/MainActivity02.kt
package kr.co.aiai.myapp

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.util.Log
import android.widget.Button
import android.widget.TextView

class MainActivity02 : AppCompatActivity() {
    lateinit var tv:TextView

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main02)
        var btn = findViewById<Button>(R.id.btn)
        tv = findViewById<TextView>(R.id.tv)
        btn.setOnClickListener{
            myclick()
        }


    }
    fun myclick(){
        var cnt = Integer.parseInt(tv.getText().toString());
        cnt--
        tv.text= cnt.toString();
    }

}